src/components/public/header-links.tsx

```
@@ -0,0 +1,40 @@
+          <span className={`relative z-10 transition-colors duration-300 ${
+            isActive 
+              ? '' 
+              : 'dark:group-hover:text-primary-light group-hover:text-primary'
+          }`}>
+            {link.displayName}
+          </span>
+          <span className={`absolute bottom-0 left-0 h-0.5 w-full origin-left bg-gradient-to-r from-blue-400 to-primary transition-transform duration-300${
+            isActive 
+              ? 'scale-x-100' 
+              : 'scale-x-0 group-hover:scale-x-100'
+          }`}></span>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)